torchao/dtypes/affine_quantized_tensor_ops.py

```diff
@@ -20,6 +20,10 @@
     _linear_int8_act_int8_weight_block_sparse_check,
     _linear_int8_act_int8_weight_block_sparse_impl,
 )
+from torchao.dtypes.uintx.gemlite_layout import (
+    _linear_fp_act_int4_weight_gemlite_check,
```

Comment:
nit: fp16? seems like gemlite only works with fp16 right now, but can be a follow up PR